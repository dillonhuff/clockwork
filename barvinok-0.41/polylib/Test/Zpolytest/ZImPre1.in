# 
M 2
3 3
2 0 0 
2 1 0
0 0 1
3 3
1 0 0 
0 1 0
0 0 1
#readDom["{i,j| 1<= i,j<=100}"]
D 1
4 4
	  1 1 0 -1
          1 0 1 -1
          1 -1 0 100
          1 0 -1 100
F 12



