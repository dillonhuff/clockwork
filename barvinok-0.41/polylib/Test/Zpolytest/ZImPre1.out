the Two zpol are equal? :1
